### PR TITLE
Label value now accept Strings and Boolean

### PR DESCRIPTION
**Pull request by trygve-lie**

### CODE REVIEW

File: lib/is.js [hunk 1]
```diff
@@ -69,6 +69,12 @@ const validValue = (value) => {
 module.exports.validValue = validValue;
 
 
+const validLabelValue = (value) => {
+    return Number.isFinite(value) || isString(value) || value === null || value === undefined;
```
Comment (digitalsadhu) [inline lib/is.js]: boolean is probably fine as well thought that could just be expressed as a string perhaps?

**Comment by trygve-lie** [comment]
What would the usecase for boolean be?

**Comment by digitalsadhu** [comment]
`success: true|false`

File: lib/is.js (hunk 1 above)
Comment (trygve-lie) [reply lib/is.js]: Now supporting boolean too.

**Pull request merged**